### ISSUE TITLE
New release to support Play 2.7

### ISSUE THREAD

**Issue by ignasi35** [webjars/webjars-play#83]
Hi @jamesward , it'd be great to have a new release compatible with Play 2.7. We've already release `2.7.0-RC3` and nearing GA (hopefully before end of year).

/cc @marcospereira

**Comment by marcospereira** [comment]
Thanks a lot for being so responsive, @jamesward!

**Comment by ignasi35** [comment]
Pff, you're so fast you make my slack discussions obsolete before anyone can respond to them.

Thanks @jamesward

**Comment by marcospereira** [comment]
@francisdb is that happening with the the combination of:

1. Play 2.7.0
2. webjars-play 2.7.0

?

**Comment by francisdb** [comment]
no, it's happening with the previous webjars play (when you forget to upgrade the webjars-play dependency)
